app/pages/transportEquipment/index/ApplyAnItemYesNoPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.transportEquipment.index
+
+import models.{Index, UserAnswers}
+import pages.sections.{ItemsSection, TransportEquipmentSection}
+import pages.{ContainerIdentificationNumberPage, QuestionPage}
+import play.api.libs.json.JsPath
+
+import scala.util.Try
+
+case class ApplyAnItemYesNoPage(equipmentIndex: Index) extends QuestionPage[Boolean] {
+
+  override def path: JsPath = TransportEquipmentSection(equipmentIndex).path \ toString
+
+  override def toString: String = "applyAnItemYesNo"
```

Comment:
We probably want cleanup logic here to remove ItemsSection - can this be done as part of this ticket or will it be done separately?